scenario-api/src/main/scala/pl/touk/nussknacker/engine/build/ScenarioBuilder.scala

```diff
@@ -47,23 +46,15 @@ class ProcessMetaDataBuilder private[build](metaData: MetaData) {
   }
 }
 
-object EspProcessBuilder {
+object ScenarioBuilder {
 
-  def id(id: String) =
+  def streaming(id: String) =
     new ProcessMetaDataBuilder(MetaData(id, StreamMetaData()))
 
-}
-
-object StreamingLiteScenarioBuilder {
-
-  def id(id: String) =
+  def streamingLite(id: String) =
     new ProcessMetaDataBuilder(MetaData(id, LiteStreamMetaData()))
 
-}
-
-object RequestResponseScenarioBuilder {
-
-  def id(id: ProcessName) =
-    new ProcessMetaDataBuilder(MetaData(id.value, RequestResponseMetaData(None)))
+  def requestResponse(id: String) =
+    new ProcessMetaDataBuilder(MetaData(id, RequestResponseMetaData(None)))
```

Comment:
should'nt these 3 method return difrrent subtypes of `ProcessMetaDataBuilder` ? 

Now it is possible to invoke e.g.`ScenarioBuilder.requestResponse("x").parallelizm(4)` which will throw an exception. It would be more straightforward if api wouldn't allow such invocation

Reply:
Probably API should be designed that way but after this change nothing is different to the previous approach - there still would be an exception becouse you could invoke ```RequestResponseScenarioBuilder
        .id(ProcessName("sample_scenario"))
        .parallelism(3)```

Reply:
maybe we can use requestResponse/streaming methods with default parameters?